.buildkite/heartbeat/heartbeat-pipeline.yml

```diff
@@ -183,9 +174,9 @@ steps:
       - label: ":ubuntu: Packaging Linux X86"
         key: "heartbeat-package-linux-x86"
         env:
-          PLATFORMS: "${PACKAGING_PLATFORMS}"
+          PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
```

Comment:
Warning: when doing the backport for `7.17` we must remove `darwin/arm64`, see https://elastic.slack.com/archives/C0522G6FBNE/p1712924637322749

Reply:
Thanks, I'm aware of that